scripts/garage

```diff
@@ -14,27 +11,32 @@ def cli():
 
 @cli.command()
 @click.option(
-    '--snapshot_dir',
-    required=True,
-    help='Directory of the pickle file to resume experiment from.')
+    '--resume_from_dir', required=True, help='Path to resume experiment from.')
 @click.option(
-    '--resume_epoch',
+    '--resume_from_epoch',
     default='last',
     help='Index of epoch to restore from. Can be "first", "last" or a number. '
     'Not applicable when snapshot_mode="last"')
-def resume(snapshot_dir, resume_epoch):
-    params = dict()
-    params['resume_from_dir'] = snapshot_dir
-    params['resume_epoch'] = resume_epoch
-    command = to_local_command(
-        params, script='garage.experiment.experiment_wrapper')
-    print(command)
-    try:
-        subprocess.call(command, shell=True, env=os.environ)
-    except Exception as e:
-        print(e)
-        if isinstance(e, KeyboardInterrupt):
-            raise
+@click.option(
+    '--log_dir',
+    default=None,
+    help='Path to save the log snapshot. If not specified, will be the same '
+    'as resume_from_dir.')
+def resume(resume_from_dir, resume_from_epoch, log_dir):
+    if log_dir is None:
+        log_dir = resume_from_dir
+
+    run_experiment(
+        run_task,
+        log_dir=log_dir,
+        resume_from_dir=resume_from_dir,
+        resume_from_epoch=resume_from_epoch)
+
+
+def run_task(_snapshot_config, _variant_data, from_dir, from_epoch):
```

Comment:
hmm perhaps just define this inside `resume()`, unless it's used elsewhere. if it is used elsewhere, make it private.